cypress/e2e/with_mock_data/systems.cy.ts

```diff
@@ -374,13 +385,33 @@ describe('Systems', () => {
         'A System with the same name already exists within the same parent System. Please enter a different name.'
       ).should('not.exist');
     });
+
+    it('displays error message if the system type id is not found', () => {
+      cy.visit('/systems');
+
+      cy.findByRole('button', { name: 'Add System' }).click();
+      cy.findByLabelText('Name *').type('Error Not found type');
```

Comment:
```suggestion
      cy.findByLabelText('Name *').type('Error type not found');
```
Picky but I think its clearer when I read it/matches with others.